### PR TITLE
HYDRA-539 : Viewport filters test

### PR DESCRIPTION
**Pull request by debloip-adsk**

### CODE REVIEW

File: test/testSamples/testViewportFilters/motionTrails.ma [hunk 1]
```diff
@@ -0,0 +1,6289 @@
+//Maya ASCII 2026ff01 scene
+//Name: motionTrails.ma
+//Last modified: Fri, May 03, 2024 03:45:46 PM
+//Codeset: 1252
+requires maya "2026ff01";
```
Comment (lilike-adsk) [inline test/testSamples/testViewportFilters/motionTrails.ma]: this needs to be removed.

File: test/testSamples/testViewportFilters/strokes.ma [hunk 2]
```diff
@@ -0,0 +1,1247 @@
+//Maya ASCII 2026ff01 scene
+//Name: strokes.ma
+//Last modified: Fri, May 03, 2024 05:44:48 PM
+//Codeset: 1252
+requires maya "2026ff01";
```
Comment (lilike-adsk) [inline test/testSamples/testViewportFilters/strokes.ma]: same here

File: test/lib/mayaUsd/render/mayaToHydra/testViewportFilters.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        super(TestViewportFilters, self).setUp()
+        # Include everything
+        cmds.modelEditor(mayaUtils.activeModelPanel(), edit=True, excludeObjectMask=0)
+
+    # This is a somewhat hacky workaround around the limitation that line width is currently (2024-05-02)
+    # not supported by mayaHydra : we stack multiple instances of the same item very close together.
+    # If we just used a single wireframe, its line width would be 1-pixel wide, which is much too thin
+    # to be reliable : by having multiple of them, it can take up more of the screen and provide more
+    # robust image comparison.
+    def stackInstances(self, itemCreationCallable, nbInstances, offset):
```
Comment (lilike-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testViewportFilters.py:80]: nice workaround

**Comment by debloip-adsk** [comment]
Good eye, thanks for catching these!

**Review by lilike-adsk — APPROVED** [review]

**Pull request merged**